statsmodels/stats/weightstats.py

```diff
@@ -226,6 +226,79 @@ def std_mean(self):
         return std / np.sqrt(self.sum_weights - 1)
 
 
+    def quantile(self, probs):
```

Comment:
I think we should make pandas returns optional, e.g. `return_pandas=True`, and return numpy array or scalar otherwise